src/Common/src/TypeSystem/Common/FieldLayoutAlgorithm.cs

```diff
@@ -101,7 +101,8 @@ public struct ComputedStaticFieldLayout
     {
```

Comment:
Nit - not your fault but it's still silly we cannot reuse the StaticBlockInfo holding exactly the same information.